sound/pulseaudio/patches/010-iconv.patch

```diff
@@ -10,6 +10,8 @@
 +if not have_iconv and cc.has_function('iconv_open')
```

Comment:
I don't think this patch is needed anymore.

Reply:
I'll check.